CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 List of the most important changes for each release.
 
+## 0.12.8
+
+### Changed or fixed
+
+- Fixed an issue where users creating accounts for themselves would be placed in the wrong facility
+- Fixed an issue where, occasionally, images would be missing from Khan Academy exercises
+- Fixed an issue where the "Usage and Privacy" modal would not close when clicking the "Close" button
```

Comment:
would be nice to avoid repeating the phrase "Fixed an issue where", mostly for style reasons.

could do something like

- Fix account creation when multiple facilities are on a device
- Prevent occasionally missing images in Khan Academy exercises
- Fixed a regression related to the "Close" button in the "Usage and Privacy" modal